### PR TITLE
DATA SCIENCE TEAM Public Transport Demand 40% completed 

### PR DESCRIPTION
**Pull request by taehwan-jung**
Week3 pull request for review from sprint1

 Use case include:
- Use case introduction
- Import data using API 2.1 version
- Handle missing values
- Preprocess for visualization
- Filter dataframes

### CODE REVIEW

**Review by Sabihulhassan98 — APPROVED** [review]
Good Work Taehwan, You can merge this pull request with the master once you get the second approval

**Review by Nish245 — APPROVED** [review]
Good Work Taehwan, All bullet points are mentioned correctly. You can merge this to the main branch

**Pull request merged**
